### PR TITLE
master node rewrite and tested with XBEE

### PR DESCRIPTION
**Pull request by Robertomendivil97**
Update to master Node for Release 1
Only receive from boat transceiver, and only work for start and kill the boat from station xbee messages

### CODE REVIEW

File: scripts/usv_master.py [hunk 1]
```diff
@@ -71,19 +86,19 @@ def main():
         elif (usvMaster.xbee_message)== "AN" or  (usvMaster.xbee_message)== "an":
```
Comment (pedrosc97) [inline scripts/usv_master.py]: Por qué usan parentesis para encapsular una variable? 
No sería mejor así?:
(usvMaster.xbee_message == "AN")

File: scripts/usv_master.py [hunk 2]
```diff
@@ -71,19 +86,19 @@ def main():
...
             current_node = subprocess.Popen("rosrun rb_missions auto_nav_position.py", shell = True)
             usvMaster.xbee_message= ""
-            print(usvMaster.xbee_message)
-            while usvMaster.mission_status != 1 and  usvMaster.stop_mission!=True and not rospy.is_shutdown():
+            while usvMaster.mission_status != 1 and  usvMaster.kill_switch!=True and not rospy.is_shutdown():
```
Comment (pedrosc97) [inline scripts/usv_master.py]: Readability: condiciones agrupadas en paréntesis.
(usvMaster.mission_status != 1) and (...) and not (...)

File: scripts/usv_master.py [hunk 3]
```diff
@@ -71,19 +86,19 @@ def main():
...
                  usvMaster.boat_current_mission_pub.publish("Atonomous Navigation")
                  rate.sleep()
             current_node.kill()
                          
         elif (usvMaster.xbee_message)== "SP" or (usvMaster.xbee_message)== "sp":
             current_node = subprocess.Popen("rosrun rb_missions speed_ch.py", shell = True)
             usvMaster.xbee_message= ""
-            while usvMaster.mission_status != 1 and usvMaster.stop_mission!=True and not rospy.is_shutdown():
+            while usvMaster.mission_status != 1 and usvMaster.kill_switch!=True and not rospy.is_shutdown():
```
Comment (pedrosc97) [inline scripts/usv_master.py]: Same as above

File: scripts/usv_master.py [hunk 4]
```diff
@@ -71,19 +86,19 @@ def main():
...
             current_node = subprocess.Popen("rosrun rb_missions auto_nav_position.py", shell = True)
             usvMaster.xbee_message= ""
-            print(usvMaster.xbee_message)
-            while usvMaster.mission_status != 1 and  usvMaster.stop_mission!=True and not rospy.is_shutdown():
+            while usvMaster.mission_status != 1 and  usvMaster.kill_switch!=True and not rospy.is_shutdown():
                  usvMaster.boat_current_mission_pub.publish("Atonomous Navigation")
                  rate.sleep()
             current_node.kill()
                          
         elif (usvMaster.xbee_message)== "SP" or (usvMaster.xbee_message)== "sp":
```
Comment (pedrosc97) [inline scripts/usv_master.py]: Same as above.

**Review by pedrosc97 — CHANGES_REQUESTED** [review]

File: scripts/usv_master.py (hunk 1 above)
Comment (vSebas) [reply scripts/usv_master.py]: Fue el rob, jaja

Comment (vSebas) [reply scripts/usv_master.py]: Ntc, va

**Review by pedrosc97 — APPROVED** [review]

**Pull request merged**